.github/workflows/ci-cvxpy.yml

```diff
@@ -0,0 +1,52 @@
+name: Integration testing with CVXPY
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    tags:
+      - '*'
```

Comment:
This has to be `'**'` to capture tag names containing a `/`. Or it could also just be `'releases/*'`.

Reply:
But why not run this on every commit? I guess if all changes come in through PR's, then this is fine, but I get lazy sometimes and commit directly to master :).

Reply:
Thanks, done